core/src/main/java/org/infinispan/persistence/manager/PersistenceManager.java

```diff
@@ -28,6 +28,11 @@
 public interface PersistenceManager extends Lifecycle {
 
    boolean isEnabled();
+
+   /**
+    * Returns whether the manager is enabled and has at least one store
+    */
+   boolean hasStore();
```

Comment:
Maybe `hasWriter` would be better, store is a bit nebulous because the loader config is also a `StoreConfiguration`.

Reply:
Sure, I am not too picky either way since it isn't public.